### PR TITLE
Add English page to the blog

### PR DESCRIPTION
**Pull request by ogwata**
確認をお願いします。

### CODE REVIEW

File: _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+title: New Year's Message and Update on Vivliostyle
```
Comment (MurakamiShinyu) [inline _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md:2]: Why not "New Year's Greetings and Updates on Vivliostyle" ?

File: _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+<div style="float: right; margin: 0 0 1em 1em;"><img src="/assets/posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle/fig1.png" alt="Implementation of named strings in Vivliostyle Viewer" style="width: 400px; box-shadow: 1px 2px 2.5px 1.5px grey;" /></div>
+
+
+## Introduction
+
+Happy New Year to you all! Thank you for Vivliostyle again this year.
+
+Last year was a “year of preparation” for Vivliostyle, as we set up our development system and launched new projects. This year, we would like to make the most of our achievements and make a big leap forward.
+
+Last year Vivliostyle has been steadily updating and adding features. Originally, We should have announced the release on this blog, but We ware unable to do so last year. So We'd like to give you a brief overview of Vivliostyle's current development status in this post.
```
Comment (MurakamiShinyu) [inline _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md:22]: "We"-->"we"?

File: _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+## Added features for vivliostyle.js
+
+First of all, the update history of [vivliostyle.js](https://github.com/vivliostyle/vivliostyle.js), which is the foundation of the entire project, can be found on the [GitHub release history page](https://github. com/vivliostyle/vivliostyle.js/releases). The following two features have been added that should not be overlooked.
```
Comment (MurakamiShinyu) [inline _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md:26]: Remove space in URL `github. com`

File: _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md (hunk 1 above)
Comment (ogwata) [reply _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md:2]: I chose "message" as it is simpler and conveys the intent better. Does this mean that "greeting" has more merit?

File: _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md (hunk 2 above)
Comment (ogwata) [reply _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md:22]: Thank you. It was an oversight.

File: _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md (hunk 3 above)
Comment (ogwata) [reply _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md:26]: This was also an oversight.

File: _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ Happy New Year to you all! Thank you for Vivliostyle again this year.
 
 Last year was a “year of preparation” for Vivliostyle, as we set up our development system and launched new projects. This year, we would like to make the most of our achievements and make a big leap forward.
```
Comment (johanneswilm) [inline _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md:20]: Unclear why "year of preparation" is in quotation marks.

Comment (johanneswilm) [reply _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md:20]: recommendation: remove quotation marks.

File: _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+Happy New Year to you all! Thank you for Vivliostyle again this year.
+
+Last year was a “year of preparation” for Vivliostyle, as we set up our development system and launched new projects. This year, we would like to make the most of our achievements and make a big leap forward.
+
+Last year Vivliostyle has been steadily updating and adding features. Originally, we should have announced the release on this blog, but we ware unable to do so last year. So we'd like to give you a brief overview of Vivliostyle's current development status in this post.
+
+## Added features for vivliostyle.js
+
+First of all, the update history of [vivliostyle.js](https://github.com/vivliostyle/vivliostyle.js), which is the foundation of the entire project, can be found on the [GitHub release history page](https://github.com/vivliostyle/vivliostyle.js/releases). The following two features have been added that should not be overlooked.
```
Comment (johanneswilm) [inline _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md:26]: "update history" == "changelog"?

Why is the update history the foundation of the project? Or do you mean vivliostyle.js? Also the phrase "First of all" seems a bit out of place. Maybe something like:

"The complete changelog of vivliostyle.js can be found on the [GitHub release history page](https://github.com/vivliostyle/vivliostyle.js/releases). The following two features are especially worth highlighting:"

File: _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+- [v2.2.0 (2020-11-26)](https://github.com/vivliostyle/vivliostyle.js/releases/tag/v2.2.0).......Add page slider in Vivliostyle Viewer UI.
+- [v2.4.0 (2020-12-28)](https://github.com/vivliostyle/vivliostyle.js/releases/tag/v2.4.0).......Support [named strings<i class="fas fa-external-link-alt"></i>](https://www.w3.org/TR/css-gcpm-3/#named-strings)for running headers and footers.
+
+One of the most promising ones is “named strings”. The following test page will allow you to see how this property can be used.
+
+- [Test cases / Named Strings](https://raw.githack.com/vivliostyle/vivliostyle.js/master/packages/core/test/files/#Named_Strings)
+
+ Click on “canary” or “stable” to launch the latest version of Vivliostyle Viewer (shown in the figure above), which implements named strings. Clicking on “2019.11” or “2018.2” will launch a previous version of Vivliostyle Viewer ( shown in the figure below ) that has not yet implemented named strings. You can thus compare this two.
```
Comment (johanneswilm) [inline _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md:35]: It is unclear whether "named strings" are in "canary" or "stable". 

"You can thus compare this two." => "This way you can compare the two."

File: _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+If you are familiar with publishing, you may nod your head and say, “Oh, you mean Running Heads”. That's true, of course, but it's not the only possibility for “named strings”. For example, it can be used in the body element, which allows you to substitute any string you specify as “named strings” in the body or headings, such as the title of a book or the name of the author.
+
```
Comment (johanneswilm) [inline _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md:38]: If you are familiar with publishing, you may recognize this feature as useful for running headers. ANd it does work for that, but it is not the only way to use "named strings".

File: _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+In addition, express our deepest gratitude Japan Business Press Co.Ltd, for their support in the development of these versions.
```
Comment (johanneswilm) [inline _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md:39]: We also want to express our deepest gratitude to Japan Business Press Co.Ltd for their support in the development of these versions.

File: _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+
+- [これからのオープンな出版標準としての「WebBook」](https://github.com/vivliostyle/community/wiki/Rapid-publishing-for-public-health-books-against-COVID-19#%E3%81%93%E3%82%8C%E3%81%8B%E3%82%89%E3%81%AE%E3%82%AA%E3%83%BC%E3%83%97%E3%83%B3%E3%81%AA%E5%87%BA%E7%89%88%E3%81%AE%E6%A8%99%E6%BA%96%E3%81%A8%E3%81%97%E3%81%A6webbook)
+
+By outputting this webbook as an intermediate file, you will be able to easily convert it to EPUB, Web pages, PDF, etc. from there. It's a dream come true!
+
+In addition, there is an ongoing discussion about the v3.0 specification at [Issue #86](https://github.com/vivliostyle/vivliostyle-cli/issues/86), so if you are interested, please join in.
+
+## And about Vivliostyle Pub
+
+Finally, let's talk about our main goal, Vivliostyle Pub. vivliostyle.js and Vivliostyle CLI 3.0 that we' ve mentioned so far are all elemental technologies of Vivliostyle Pub. Its requirements specification is summarized below.
```
Comment (johanneswilm) [inline _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md:57]: "we' ve" => "we've" or "we have"

**Review by johanneswilm — CHANGES_REQUESTED** [review]

File: _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md (hunk 4 above)
Comment (ogwata) [reply _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md:20]: I see. I'll fix it.

File: _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+  - Vivliostyle Core
+  - Vivliostyle Pub
+  - Vivliostyle CLI
+---
+<div style="float: right; margin: 0 0 1em 1em;"><img src="/assets/posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle/fig1.png" alt="Implementation of named strings in Vivliostyle Viewer" style="width: 400px; box-shadow: 1px 2px 2.5px 1.5px grey;" /></div>
+
+
+## Introduction
+
+Happy New Year to you all! Thank you for Vivliostyle again this year.
```
[the remark this answers is not in the document]
Comment (ogwata) [reply _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md:18]: It is indeed an odd sentence. Your suggestion is accurate, but I want to thank the user as well, so how about the following sentence?

> Vivliostyle would like to thank all the contributors who have collaborated with us on our projects over the last year and all the users of our products.

File: _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md (hunk 5 above)
Comment (ogwata) [reply _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md:26]: Thanks for the precise suggestion. How about changing it as follows.

> First, let's talk about vivliostyle.js, the foundation of the entire project. The complete changelog of vivliostyle.js can be found on the GitHub release history page. The following two features are especially worth highlighting:

File: _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md (hunk 6 above)
Comment (ogwata) [reply _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md:35]: Thanks.

“Click on “canary” or “stable”” => “Click on “canary” and “stable””

> "You can thus compare this two." => "This way you can compare the two."

I see. I'll fix it.

File: _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md (hunk 7 above)
Comment (ogwata) [reply _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md:38]: I see. I'll fix it.

File: _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md (hunk 5 above)
Comment (johanneswilm) [reply _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md]: looks good

File: _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (johanneswilm) [reply _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md]: Ok, yes, this sounds good.

File: _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md (hunk 8 above)
Comment (ogwata) [reply _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md:39]: I see. I'll fix it.

File: _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md (hunk 9 above)
Comment (ogwata) [reply _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md:57]: I see. I'll fix it.

File: _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md [hunk 11]
```diff
@@ -32,9 +32,9 @@ One of the most promising ones is “named strings”. The following test page w
 
 - [Test cases / Named Strings](https://raw.githack.com/vivliostyle/vivliostyle.js/master/packages/core/test/files/#Named_Strings)
 
- Click on “canary” or “stable” to launch the latest version of Vivliostyle Viewer (shown in the figure above), which implements named strings. Clicking on “2019.11” or “2018.2” will launch a previous version of Vivliostyle Viewer ( shown in the figure below ) that has not yet implemented named strings. You can thus compare this two.
+ Click on “canary” and “stable” to launch the latest version of Vivliostyle Viewer (shown in the figure above), which implements named strings. Clicking on “2019.11” or “2018.2” will launch a previous version of Vivliostyle Viewer ( shown in the figure below ) that has not yet implemented named strings. You can thus compare this two.
```
Comment (johanneswilm) [inline _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md:35]: ok, so the "named strings" are both in canary and in stable? In that case revert to "or and make the entire sentence something like:  

Click on “stable” or "canary" to launch the latest versions of Vivliostyle Viewer (shown in the figure above). Both of them contain the "named strings" feature. Clicking on “2019.11” or “2018.2” will launch an older version of Vivliostyle Viewer (shown in the figure below) without said "named strings" feature. This way you can compare versions of Vivliostyle with or without the "named strings" feature.

Comment (ogwata) [reply _posts/2021-01-06-new-years-greetings-and-updates-on-vivliostyle.md:35]: I see! I'll fix it that way.

**Review by MurakamiShinyu — APPROVED** [review]

**Pull request merged**